docs/source/cell.rst

```diff
@@ -14,7 +14,7 @@ in the case Cell segmentation data was collected as part of this experiment.
 These are properties that are shared by all bright Spots and
 Traces that belong to an individual Cell. Each row in the table
 corresponds to a different Cell studied in the experiment and is
-identified by a unique Cell_ID that links the data reported in this
+identified by a unique **Cell_ID** that links the data reported in this
 table with data stored in one of the other tables (i.e.,
 :ref:`core`, :ref:`trace`, etc.,). The header should note
```

Comment:
Is the reference to Trace data table appropriate here? Cell_ID is mentioned in the core table but not in the current trace table.

Reply:
I see what you mean. I removed reference to trace adn instead added a reference to subcell and mapping in addition I also saw a problem in subcell and fixed it. Removed Cell_ID in the example because is redundant to the mandatory Sub_Cell_ROI_ID to Cell_ROI mapping in the subcell table.

Reply:
Same issue found and fixed in Trace table

Reply:
I also saw a problem in mapping example and fixed it. 
Removed Cell_ID in the example because is redundant to the mandatory Sub_Cell_ROI_ID to Cell_ROI mapping in the subcell table.

Reply:
Similar problem in subcell and extracell. Fixed it